src/app/shell/Header.tsx

```diff
@@ -26,50 +26,50 @@ import ExternalLink from '../dim-ui/ExternalLink';
 const destiny1Links = [
   {
     state: 'destiny1.inventory',
-    text: 'Header.Inventory'
+    text: t('Header.Inventory')
```

Comment:
This ends up using `t` before translations are loaded, resulting in an empty string.